frontend/src/components/story/StoryPeriod.vue

```diff
@@ -20,30 +22,49 @@ export default {
   },
   data() {
     return {
+      sortedDays: [],
```

Comment:
Wouldn't this be way easier using a `computed` property? You can still trigger a reload in your watcher, but as far as I understand there is no need to manually manage this state, or is there? This way I think the `nextTick` is also unnecessary